public/html/popup.html

```diff
@@ -1,12 +1,13 @@
 <!doctype html>
-<html class="mode__popup light">
+<html class="mode__popup light splash-screen">
```

Comment:
We already have a class added `light` or `dark`, maybe we can do this all with just CSS? 

So if the class is `dark` set the `background-color: #12100F`?

Reply:
I thought on this and decided to completely remove this class, and instead of this add absolute `splash-screen` div